src/Symfony/Component/Translation/Dumper/FileDumper.php

```diff
@@ -109,6 +111,8 @@ protected function formatCatalogue(MessageCatalogue $messages, $domain, array $o
      * @param string           $domain
      *
      * @return string representation
+     *
+     * @deprecated since version 2.8, to be removed in 3.0. Overwrite formatCatalogue() instead.
      */
     abstract protected function format(MessageCatalogue $messages, $domain);
```

Comment:
@aitboudad this method should be implemented now (throwing an exception saying it must be implemented), to allow new child classes to implement only ``formatCatalogue`` and not both methods